lib/loadGltfUris.js

```diff
@@ -15,21 +15,11 @@ function loadGltfUris(gltf, basePath, callback) {
     async.each(['buffers', 'images', 'shaders'], function(name, asyncCallback) {
             loadURI(gltf, basePath, name, asyncCallback);
         }, function(err) {
-            if (err) {
-                if (callback) {
-                    callback(err);
-                }
-                else{
-                    throw err;
-                }
-            }
-            else if (callback) {
-                callback();
+            if (defined(callback)) {
```

Comment:
`combinePrimitivesSpec` has a couple specs that run `loadGltfUris` without a callback. I'll see if I can change these.